nixos/modules/services/networking/dnscrypt-proxy2.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }: with lib;
+
+let
+  cfg = config.services.dnscrypt-proxy2;
+
+  json = pkgs.writeText "dnscrypt-proxy.json" (builtins.toJSON cfg.config);
+
+  toml = pkgs.runCommand "dnscrypt-proxy.toml" {} ''
+    ${pkgs.json-to-toml}/bin/json-to-toml < ${json} > $out
```

Comment:
I think it make sense to just re-use `remarshal` instead. It does the same thing and if we agree on one tool, it can be re-used for different services. See also in the telegraf, influxdb and gitlab-runner.

Reply:
I guess so. Both implementations are not terrible complex.

Reply:
Btw. I can recommend nixops for you use case independent from your pr. Our nix builder also handles armv7l for that matter.

Reply:
It would be nice if the output is pretty-printed with indentation to make it more readable.